src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1689,7 +1689,7 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>The first operand of an overloaded shift operator must have the same type as the containing type, and the type of the second operand must be int</value>
   </data>
   <data name="ERR_InterfacesCantContainConversionOrEqualityOperators" xml:space="preserve">
-    <value>Interfaces cannot contain conversion, equality, or inequality operators</value>
+    <value>Conversion, equality, or inequality operators declared in interfaces must be abstract</value>
```

Comment:
It seems like the implementation disallows conversion operators in interfaces whether they are abstract or not. Is this expected to change in a subsequent PR?

Reply:
> It seems like the implementation disallows conversion operators in interfaces whether they are abstract or not. Is this expected to change in a subsequent PR?

Yes, working on that right now. I felt that it should be fine to go with the final wording of the message.